packages/vote/net/query.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package net;
+
+option go_package = ".;net";
+
+service VoterQuery {
+    rpc Opinion (QueryRequest) returns (QueryReply) {}
+}
+
+message QueryRequest {
+    repeated string id = 1;
+}
+
+message QueryReply {

```

Comment:
Just as a reminder, we will eventually need to sign each `QueryReply` and verify its validity on the receiving side.